packages/marshal/marshal.js

```diff
@@ -4,7 +4,7 @@
 /// <reference types="ses"/>
 
 import Nat from '@agoric/nat';
-import { assert, details } from '@agoric/assert';
+import { assert, details as d, q } from '@agoric/assert';
```

Comment:
I’m surprised this isn’t `quote as q`.

Reply:
Unfortunately the `agoric-sdk` assert module does not also export it as `quote`, which is a bad oversight in my previous PR where I tried to prepare `agoric-sdk` better for switching the SES-shim's modern assert module.

In any case, I clearly should not make this change to agoric-sdk in this PR. Resolving.